metricbeat/k8s-dev-tools/Dockerfile.debug

```diff
@@ -4,10 +4,10 @@ ENV PATH=/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/go/bin:/usr/local/go/bin
 
 ENV CGO_ENABLED=0
 
-RUN apk add --no-cache git
+# RUN apk add --no-cache git
 RUN go install github.com/go-delve/delve/cmd/dlv@v1.8.3
 
-COPY metricbeat /usr/share/metricbeat/metricbeat
+COPY build/metricbeat /usr/share/metricbeat/metricbeat
 
 FROM alpine:3.15
 
```

Comment:
is it required to set env variables:
```
ENV ELASTICSEARCH_PASSWORD=changeme
ENV ELASTICSEARCH_USERNAME=elastic
ENV ELASTICSEARCH_HOST=elasticsearch
```
in dockerfile? those will be provided in manifest, not?

Reply:
I thought about those to make it easier to run as a docker container instead of inside kubernetes. I believe that's not necessary anymore